java/src/test/java/com/unascribed/flexver/Ordering.java

```diff
@@ -0,0 +1,52 @@
+package com.unascribed.flexver;
+
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
+public enum Ordering {
+    Less("<"),
+    Equal("="),
+    Greater(">");
```

Comment:
Conventionally, enum constants (all constants, in fact) are SCREAMING_SNAKE_CASE.

```suggestion
    LESS("<"),
    EQUAL("="),
    GREATER(">");
```
(Suggestion must be applied via IDE refactor, not in GitHub web.)

Reply:
That's pretty embarrassing. Guess I've been doing too much Rust lately